cns/kubeconfigtemplate.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
It gets modified by [this script](https://github.com/Azure/azure-container-networking/blob/master/npm/examples/windows/setkubeconfigpath.ps1#L4)

Reply:
I'm not seeing where we run that script in CNS?
It looks like the script just inserts the in-cluster config values in to this template anyway, so I'm not sure what the difference in result is. If we do really need to modify the config, the `rest.Config` is mutable and we should probably do this in code rather than by some sideload templating.

Reply:
it's being run before starting CNS in yml

![image](https://user-images.githubusercontent.com/13284103/167714230-193512b7-8b2c-44b5-8a64-eeb0ff931473.png)